django/db/migrations/migration.py

```diff
@@ -162,8 +162,10 @@ def unapply(self, project_state, schema_editor, collect_sql=False):
                 schema_editor.collected_sql.append("--")
                 if not operation.reduces_to_sql:
                     continue
-            if not schema_editor.connection.features.can_rollback_ddl and operation.atomic:
-                # We're forcing a transaction on a non-transactional-DDL backend
+            atomic_operation = operation.atomic or (self.atomic and operation.atomic is not False)
```

Comment:
This is the same condition as in f91a04621ea8d7a657ff755645d823026257e898 and a test fails if this condition is changed there but not if this one is changed, so maybe a test case is missing here?